client/src/components/_common/Icon/Icon.jsx

```diff
@@ -2,27 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Icon.module.css';
 
-const Icon = ({ children, className, name }) => {
+const Icon = ({ children, className, dataTestid, name }) => {
```

Comment:
**Explanation:** So `<Button />` tests can distinguish between the two icons it can (simultaneously) have.